package.json

```diff
@@ -8,14 +8,16 @@
   },
   "scripts": {
     "build": "webpack --mode production",
-    "start": "npm run build && node src/server/index.js",
+    "start": "node src/server/index.js",
     "client": "webpack-dev-server --mode development --devtool inline-source-map --hot --port 3001",
     "server": "PORT=8080 nodemon src/server/index.js",
-    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "dev": "concurrently \"yarn server\" \"yarn client\"",
     "lint": "yarn eslint 'src/**/*.js' 'test/**/*.js'",
     "test:server": "NODE_ENV=test ANVIL_API_KEY=test mocha 'test/server/**/*.test.js' --require test/server/environment.js",
     "test:client": "NODE_ENV=test mocha 'test/client/**/*.test.js' --require test/client/environment.js --require @babel/register",
-    "test": "yarn lint && yarn test:server && yarn test:client"
+    "test": "yarn lint && yarn test:server && yarn test:client",
+    "gcp-build": "yarn build",
+    "deploy": "gcloud app deploy --project=$ETCH_DEMO_PROJECT"
```

Comment:
these two scripts are only needed for deploying to App Engine and are used in the Deploy workflow